backstop.json

```diff
@@ -2102,6 +2102,20 @@
       "misMatchThreshold" : 0.1,
       "onBeforeScript": "",
       "onReadyScript": ""
+    },
+    {
+      "label": "input:disabled",
+      "url": "http://localhost:3000/section/1.22.1-2/fullscreen",
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "selectors": [
+        ".object-section"
+      ],
+      "readyEvent": "styleguideRendered",
+      "delay": 20,
+      "misMatchThreshold" : 0.1,
+      "onBeforeScript": "",
+      "onReadyScript": ""
```

Comment:
I’ve updated this PR with the shorter backstop scenario setup, but of course that meant copying the properties shifted out of the scenarios & into the global config — there may be conflicts on merge.